qemu/tests/cfg/block_hotplug_passthrough.cfg

```diff
@@ -0,0 +1,16 @@
+- block_hotplug_passthrough:
+    type = block_hotplug_passthrough
+    only virtio_scsi
+    create_command = "modprobe -r scsi_debug; modprobe sg; modprobe scsi_debug add_host=1 dev_size_mb=2048"
+    images += " stg0"
+    boot_drive_stg0 = no
+    drive_bus_stg0 = 1
+    drive_format_stg0 = scsi-block
+    image_raw_device_stg0 = yes
+    image_format_stg0 = raw
+    stg_image_size = 2G
+    Linux:
+        dd_test = "dd if=/dev/zero of=/dev/%s bs=1M count=40 oflag=direct; dd if=/dev/%s of=/dev/null bs=1M count=40 iflag=direct"
+    Windows:
+        driver_name = vioscsi
+        iozone_options = '-az -b c:\{0}_iozone.xls -g 1g -y 32k -i 0 -i 1 -f {0}:\test-data'
```

Comment:
Suggest  `iozone_options = '-az -b c:\{0}_iozone.xls -g 1g -y 32k -i 0 -i 1 -I -f {0}:\test-data'` ,` -I` options that do io directly

Reply:
updated, thanks~